src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -26,8 +26,8 @@ ov::runtime::Tensor generate(const std::shared_ptr<ov::Node>& node,
     size_t inNodeCnt = node->get_input_size();
     InputGenerateData inGenData;
     if (elemType.is_real()) {
-        inGenData.range = 2560;
```

Comment:
As I remember, we used such ranges for bf16/f16... @maxnick What do you think?

Reply:
The whole idea of using the range of 2560 and the resolution 256 is to generate such input data where all the floating point values have mantissa with bits set in the first 7 binary digits, so that to avoid input and output data rounding when fp32 is converted to bf16. In other words, with such parameters of the old data generation routine, all the input fp32 values were directly converted to bf16 as is, thus helping to avoid results difference between runs on the CPU plugin with bf16 precision and fp32 reference.  Also this problem was explained in: https://github.com/openvinotoolkit/openvino/pull/14691#discussion_r1104391322

Reply:
@maxnick resolution changed back to 256